cheatsheets/gleam-for-elixir-users.md

```diff
@@ -659,3 +659,32 @@ pub fn main() {
 }
 ```
 
+## Patterns
+
+Same as Elixir, Gleam has pattern matching, which is used for matching complex structured data. In Gleam we use `as` to name the variable, same as using `=` in Elixir.
+
+#### Gleam
+
+```gleam
+let list = [1, 2, 3]
+
+let assert [first as 1, second .. rest] = list
+
+let assert Foo(foo: 10 as foo, bar: 20 as bar) = Foo(foo: 10, bar: 20)
```

Comment:
No "foo" please, it's a rather depressing WW2 reference.

Reply:
I did not know till now, no more foo/foobar anymore for me then.

Reply:
Sure, thanks for suggesting